WordPress/openverse
# Add `catalog/init` just command

## Fixes
An inconvenience of having to turn up the whole stack for testing catalog-only changes.

## Description


This PR adapts the `load_sample_data.sh` script to accept an option for only loading the upstream DB data. Luckily the script was modular enough to make this a pretty easy change. Kudos to @dhruvkb here! This way the API command can continue working as usual.

## Testing Instructions

Run the new command and try querying the upstream DB to see it's filled. This process returns zero results in `main`.

```sh
just down -v && just  catalog/init
# or
just catalog/recreate

# then verify with...
just catalog/pgcli
```

```sql
SELECT COUNT(*) FROM image;
```

```sql
-- Result must be:
+-------+
| count |
|-------|
| 5000  |
+-------+
SELECT 1
```

Also, confirms the `just recreate` and `just api/recreate` commands continue working as expected.

## Checklist


- [N/A] I added or updated tests for the changes I made (if applicable).
- [n/a] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
